pom.xml

```diff
@@ -99,14 +99,14 @@
     <module>sql/hive</module>
     <module>unsafe</module>
     <module>assembly</module>
-    <module>external/twitter</module>
-    <module>external/flume</module>
-    <module>external/flume-sink</module>
-    <module>external/flume-assembly</module>
-    <module>external/mqtt</module>
-    <module>external/mqtt-assembly</module>
-    <module>external/zeromq</module>
-    <module>examples</module>
+    <!--<module>external/twitter</module>-->
```

Comment:
Well, as I say, there's no way this PR can be merged like this anyway, so please close it?